core/templates/components/ck-editor-helpers/ck-editor-copy-toolbar/ck-editor-copy-toolbar.component.spec.ts

```diff
@@ -81,6 +81,12 @@ describe('CkEditor copy toolbar', () => {
     expect(blurSpy).toHaveBeenCalled();
   });
 
+  it('should revert cursor to normal when modal is closed', () => {
+    component.ngOnDestroy();
```

Comment:
do a setup fist ii.e, set the cursor to copy! Maybe have this structure:
1. activate copy mode
2. assert cursor to be copy
3. call ngDestroy
4. expect the cursor to be '';

Reply:
Sorry for that, I'm slightly new to writing tests. I have made the changes as you suggested - do have a look.

Reply:
The tests were good we just tried to make it better and specific! I'll take a pass now!